packages/data-explorer-ui/src/components/common/Banner/banner.styles.ts

```diff
@@ -0,0 +1,16 @@
+import styled from "@emotion/styled";
+import { Banner } from "./banner";
+
+export const BannerPrimary = styled(Banner)`
+  background-color: ${({ theme }) => theme.palette.primary.main};
+  color: ${({ theme }) => theme.palette.common.white};
+  padding: 4px 8px;
```

Comment:
Let's make the padding 8px all around (I see Figma doesn't specify paddingX but let's make each side 8px).

Reply:
Hey @frano-m, I made this change. The height updated to `40px` with this however. The Figma has it set to `32px`. Shouldn't be an issue?

Reply:
Updated this. A `line-height` was added.